src/Divider/Divider.js

```diff
@@ -22,7 +23,12 @@ export const styleSheet = createStyleSheet('Divider', (theme) => {
   };
 });
 
-export default function Divider(props, context) {
+type Props = {
+  absolute?: boolean,
+  className?: string,
+};
+
+export default function Divider(props:Props, context:{styleManager: Object}):Element {
```

Comment:
FYI Looks like if we add this plugin we'll get this check by default:

https://github.com/gajus/eslint-plugin-flowtype

Have you ever tried this plugin? Is it worth trying?